### DISCUSSION TITLE
How to use Gitea as a mirror for GitHub

### DISCUSSION THREAD

**Discussion by TMUniversal** [cooperspencer/gickup#127] [Q&A] [answered]
## How can I use the Gitea backup destination?

- Can I mirror organizations to a backup destination?
- How do I back up wikis?
- Can I have gickup create organizations for every organization it is supposed to back up?

**Comment by cooperspencer** [comment]
as far as I remember that was implemented with https://github.com/cooperspencer/gickup/pull/128

**Comment by TMUniversal** [comment, marked as answer]
I hadn't realized this was still open.
If anyone else needs this, please see https://github.com/cooperspencer/gickup/issues/126#issuecomment-1344482595 for details.
